README.md

```diff
@@ -869,124 +871,216 @@ After creating an initialization request successfully you can check at which sta
 }
 ```
 
-## ZOOP INCOME TAX RETURN (ITR) GATEWAY (Beta)
+## ZOOP INCOME TAX DEPARTMENT V2.0 (ITD) GATEWAY (Beta)
 
-<a name="itrIntro"></a>
+<a name="itdIntro"></a>
 
 ### 1. Introduction
 
-Income Tax Return is the form in which assessee files information about his Income and tax thereon to Income Tax Department. The Zoop ITR Gateway allows allows you to fetch the tax returns of your client and make better business decision on them.
+Income Tax Department holds and verifies the earnings and tax related information of an individual.
+Income tax return is a form where a person submits information about his income earned and the tax paid to the Department. Whereas 26AS is a consolidated annual tax credit statement of an individual's earnings as per the ITD records. The Zoop ITD Gateway allows you to fetch and verify the financial information of your client in order to make better business decisions.
 
-<a name="itrProcessFlow"></a>
+<a name="itdProcessFlow"></a>
 
 ### 2. Process Flow
 
-1. At your backend server, Initiate the ITR transaction using a simple POST Rest API call. Details of these are available in the documents later. You will require API key and Agency Id for accessing this API which can be generated from the Dashboard. This gateway transaction id then needs to be communicated back to the frontend to our Web SDK.
+1. At your backend server, Initiate the ITD transaction using a simple POST Rest API call. Details of these are available in the documents later. You will require API key and Agency Id for accessing this API which can be generated from the Dashboard. This gateway transaction id then needs to be communicated back to the frontend to our Web SDK.
 2. This gateway transaction id then needs to be communicated back to the frontend to our Web SDK.
-3. After adding the Web SDK in your project, client has to pass the above generated transaction id to an SDK function, called `zoop.initIncomeTaxReturns(gatewayOption)` which is to help us to under different requirement of the gateway you are trying to open.
-4. After the initialization of the transaction, to open the gateway, you have to call `zoop.openIncomeTaxReturnsGateway(<<transaction_id>>)` function.
+3. After adding the Web SDK in your project, client has to pass the above generated transaction id to an SDK function, called `zoop.initIncomeTaxDepartment(gatewayOption)` which is to help us to under different requirement of the gateway you are trying to open.
```

Comment:
@iAziz786 We should make this `zoop.initIncomeTaxDepartmentGateway(gatewayOption)` instead.